### PR TITLE
Adding cran-comments.md file

### PR DESCRIPTION
**Pull request by heemanshusuri**

### CODE REVIEW

File: cran-comments.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+## Test environments
+* local: windows-x86_64-devel
+* azure devops: Ubuntu:16.04 LTS
+* r-hub: windows-x86_64-devel, ubuntu-gcc-release
+* win-builder: windows-x86_64-devel
+
+## R CMD check results
+0 errors | 0 warnings | 1 note
+
+There is 1 note about a "Found the following hidden files and directories: .azure-pipelines and .github". The .azure-pipelines directory contains the pipelines to run against our code. The .github directory contains templates for reporting a bug or requesting a new feature.
```
Comment (ninghu) [inline cran-comments.md]: Shall we add files under .github and .azure-pipeline to .Rbuildignore?

Comment (heemanshusuri) [reply cran-comments.md]: good point. will add it

**Comment by ninghu** [comment]
Remember to create a release tag ("v0.5.2") after this change is in.

File: cran-comments.md (hunk 1 above)
Comment (ninghu) [reply cran-comments.md]: Ln12 can be removed.

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# azureml (0.5.2)
```
Comment (ninghu) [inline NEWS.md]: azureml (v0.5.2)

**Pull request merged**